server/src/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -461,7 +463,12 @@ private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO
             param.setClientTunnelSession(parsedHostInfo.third());
         }
 
-        sb.append("/ajax?token=" + encryptor.encryptObject(ConsoleProxyClientParam.class, param));
+
+        if(webSocketRequest){
```

Comment:
Syed decided to keep both ajax and novnc working as of now. So have kept it, will remove it once novnc is complete and tested on all hypervisors